index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+    <title>My Project</title>
+</head>
+
+<body>
+    <header>
+        <div class="logoIdea">WF</div>
+        <nav>
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="top-section1">
+            <h1> Weather Forecast</h1>
+            <h2>San Francisco, CA</h2>
+            <div id="weather-chart"></div>
+            <!-- <canvas id="weather-chart" width="400" height="200"></canvas> -->
+        </section>
+    </main>
+    <footer>
+
+    </footer>
+
+    <div class="weather-dashboard"></div>
+
+
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+    <script src="/index.js"></script>
```

Comment:
The code doesn't get loaded.  / is the root of the entire file system. ./ works as does leaving off the slash.